src/utils/array.js

```diff
@@ -0,0 +1,10 @@
+export function* range(from, to) {
```

Comment:
It's just a github highlighting, an interpreter detects it as a variable. But I'll change it, thx